br/pkg/restore/db.go

```diff
@@ -43,6 +43,19 @@ func NewDB(g glue.Glue, store kv.Storage) (*DB, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+
+	if len(policyMode) != 0 {
+		// Set placement mode for handle placement policy.
+		err = se.Execute(context.Background(), fmt.Sprintf("set @@tidb_placement_mode='%s';", policyMode))
+		if err != nil {
+			// There are two reason reach here
+			// if tidb version doesn't support placement mode, we can just ignore it.
+			// if set failed. the default mode in tidb is 'STRICT', so the behaviour works as default.
```

Comment:
It's better check if the error is `ErrUnknownSystemVariable` and skip it. Other errors should be returned.

Reply:
fixed.